src/pages/Recipes/api.ts

```diff
@@ -0,0 +1,14 @@
+import { BASE_URL } from "../../config";
+
+export type FetchAllRecipesResponse = {
+  recipes: {
+    id: string;
+    name: string;
+    author_id: string;
+  }[];
+};
+
+export const fetchAllRecipes = async (): Promise<FetchAllRecipesResponse> => {
+  const response = await fetch(`${BASE_URL}/recipes`);
```

Comment:
Nothing to change. One suggestion but depends on the project type we can ignore it. It would be nice to create resources out of the API endpoints. Can slowly become JS api sdk too. ie. Here we can have Recipes resource so that we can be agnostic of how we fetch it from API

In this file
```
  const response = await new Recipes().get()
```

In resource file. Recipes.ts


```
export default class Recipes extends BaseResource // or model
{
    

     static resourceUrl() {
         return `${BASE_URL}/recipes`;
     }          

      get(params) {
         // Can define fetch or axios adapters here or in base resource
         // and can change them anytime without worrying changing many places
         return this.fetch(Recipes.resourceUrl());
    }
}

```

Reply:
I think that could work, I was mostly trying to replicate the TK style, curious to hear what the department opinion is on this.